tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/BinaryProjectsImportWizardPage.java

```diff
@@ -54,6 +48,14 @@
 import org.eclipse.swt.widgets.Text;
 import org.eclipse.wst.common.frameworks.datamodel.IDataModel;
 
```

Comment:
Also in the resources folder is an "importorder" file that shows our preferred import ordering, so please go ahead and import that file too.